modules/registry-third-party-registries.adoc

```diff
@@ -14,3 +14,30 @@ creation.
 Refreshing the fetched tags is as simple as running `oc import-image
 <stream>`. When new images are detected, the previously described build and
 deployment reactions occur.
+
+[id="authentication_{context}"]
+== Authentication
+{product-title} can communicate with registries to access private image
+repositories using credentials supplied by the user. This allows {product-title}
+to push and pull images to and from private repositories.
+
+[id="registry-authentication_{context}"]
+=== Registry authentication with Podman
+Some container image registries require access authorization. Podman is an open source tool for managing containers and container images and interacting with image registries. You can use Podman to authenticate your credentials, pull the registry image, and store local images in a local file-system. The following is a generic example of authenticating the registry with Podman. You can use the link:https://catalog.redhat.com/software/containers/explore[Red Hat Ecosystem Catalog] to search for specific container images from the Red Hat Repository. Select the image you need. Then click the *Get this image* tab to find the `podman` command for your container image.
```

Comment:
I would suggest change file-system >> file system

Reply:
After this statement,>> The following is a generic example of authenticating the registry with Podman.  
_Suggestion_
1. Combine these two as step 1 (You can use the link:https://catalog.redhat.com/software/containers/explore[Red Hat Ecosystem Catalog] to search for specific container images from the Red Hat Repository. Select the image you need. )
2. Step 2>> Click *Get this image* to find the command for your container image:

The add the existing steps.

Reply:
Addressed